src/App.tsx

```diff
@@ -51,6 +52,7 @@ function App() {
                 <ReactCustomScrollbars autoHeight autoHeightMin={'100vh'} autoHide>
                     <Routes>
                         <Route path={'/'} element={<MainPage/>} />
+                        <Route path={'/contact'} element={<ContactPage/>} />
```

Comment:
You don`t need it here, we belove iterate over `menuItems`, so it enough when it is there

Reply:
done, fixed in other files too